smclient/smclient.go

```diff
@@ -770,6 +771,14 @@ func getTimeFromPB(timePB *timestamp.Timestamp) *time.Time {
 	return nil
 }
 
+func timeToPB(time *time.Time) *timestamp.Timestamp {
```

Comment:
Why do you need this wrapper? Use  `timestamppb.New(time.Now())`

Reply:
To handle case when time is empty.
Do you suggest to simply use new (it returns not nil in param is default initialized)?

Reply:
Timestamp in monitoring not optional so no need to handle nil case.